src/plugins/subsystems/drm/drm.sh

```diff
@@ -349,9 +372,65 @@ function get_supported_mode_per_connector()
   printf '%s\n' "$modes"
 }
 
+# Creates a binary EDID file, for each connected monitor, on the current folder.
+function get_edid()
+{
+  declare -a monitors
+
+  monitors=( $(ls -d /sys/class/drm/*/edid) )
+
+  say ""
+
+  if [ ${#monitors[@]} -lt 1 ]; then
+	  complain "Could not find any listed monitors!"
+  fi
+
+  for i in ${monitors[@]};
+  do
+          echo $i | awk -F"/" '{print "Wrote: " $5".bin"}'
+	  cat $i > $(echo $i | awk -F"/" '{print $5}')".bin"
+  done
+
+  say ""
+  
+}
+
+# Decodes selected EDID binary file. Filename must match exactly
+# the same path under sysfs, to prevent decoding non-EDID binary files.
+# e.g card0-DP-1.bin is valid for /sys/class/drm/card0-DP-1/edid
+function decode_edid ()
+{
+  declare -a monitors
+  declare -a files
+  declare -a edids
+
+  monitors+=( $(ls -d /sys/class/drm/*/edid | awk -F"/" '{print $5".bin"}') )
+  files+=( $(ls | grep bin) )
+
+  edids+=( $(echo ${monitors[@]} ${files[@]} | tr ' ' '\n' | sort | uniq -d) )
+
+  if [ ${#edids[@]} -lt 1 ]; then
+	  complain "Could not find any valid EDID binary files!"
+	  warning "Run kw drm --get-edid first!"
+	  say""
+	  return
+  fi
+
+  if [ ${#edids[@]} -gt 1 ]; then
+	  for i in `seq 0 1 $((${#edids[@]}-1))`; do
+		  say $i". "${edids[$i]}
+	  done
+          selection=$(ask_with_default "Decode which file: [0-$((${#edids[@]}-1))]" "0")
+	  edid-decode < ${edids[$selection]}
```

Comment:
`edid-decode` it is not part of the kw dependencies, you have to add it for the deploy and the local case.